### ISSUE TITLE
The code about SAT

### ISSUE THREAD

**Issue by LANMNG** [MegviiDetection/video_analyst#52]
Hi, @lzx1413, this is great project! Thank you so much!  By the way,  When will you open the source code about SAT? I am very interesting in this amazing job. I'm looking forward to seeing the code soon

**Comment by lzx1413** [comment]
Thanks for your attention, because we complete the SAT with an internal framework, it will take some time to migrate pytorch. We will release the test code in this week, please keep tuned.

**Comment by lzx1413** [comment]
@LANMNG  the sat_pipeline and davis tester have been released with transferred model params. Please refer to https://github.com/MegviiDetection/video_analyst/blob/master/tools/test/test_DAVIS2017.sh and related instruction in docs

**Comment by LANMNG** [comment]
@lzx1413 Thank you so much! This is an amazing work!

**Comment by LANMNG** [comment]
@lzx1413 Excuse me, I am so sorry to bother you, I wonder when will you release the training code of SAT?

**Comment by lzx1413** [comment]
@LANMNG  we have finished the training code for SAT,  but there is still a  minor gap of performance (about 2 points )between our internal framework and pytorch. We are refining the hyper-params for pytorch in the training phase now and code will be released in this week.

**Comment by LANMNG** [comment]
@lzx1413 thank you so much! I expect it with my pleasure

**Comment by lzx1413** [comment]
@LANMNG  we have released a alpha version of sat training code.

**Comment by LANMNG** [comment]
@lzx1413 I get it. Thank you!

**Comment by LANMNG** [comment]
@lzx1413 I encount a problem during the test process, the final saved images are all black in the results_multi folder, i don`t know what's the use of all these images (mask)

**Comment by lzx1413** [comment]
Because the saved mask are matrixes whos elements are instance labels such as 0,1,2. You can see the results by enable save_video in the config file.

**Comment by LANMNG** [comment]
@lzx1413 I meet a question, in video_analyst/videoanalyst/data/utils/crop_track_pair.py line 71, why scale_curr_ = z_size / sc_, it may should be scale_curr_ = x_size / sc_ ?
